sql/keys.go

```diff
@@ -57,7 +57,11 @@ var normalize = unicode.SpecialCase{
 // NormalizeName normalizes to lowercase and Unicode Normalization Form C
 // (NFC).
 func NormalizeName(name string) string {
-	return norm.NFC.String(strings.Map(normalize.ToLower, name))
+	lower := strings.Map(normalize.ToLower, name)
```

Comment:
with a string-adapted LRU cache, wouldn't you be able to get down to mostly zero (I'm assuming that NormalizeName isn't used on user data, just on schema and the like)?